pipeline/tests/asm.rs

```diff
@@ -341,22 +342,22 @@ fn read_poly_files() {
         // generate poly files
         let mut pipeline = Pipeline::<Bn254Field>::default()
             .from_file(resolve_test_file(f))
-            .with_output(tmp_dir.to_path_buf(), true);
+            .with_output(tmp_dir.to_path_buf(), true)
+            .with_backend(BackendType::EStarkDump);
```

Comment:
why with backend EStarkDump?

Reply:
Because no EStark files are being written by default now. If you want them, you have to use the `EStarkDump` backend.